cmd/utils/flags.go

```diff
@@ -1442,6 +1448,11 @@ func setTxPool(ctx *cli.Context, cfg *blockchain.TxPoolConfig) {
 	if ctx.GlobalIsSet(TxPoolLifetimeFlag.Name) {
 		cfg.Lifetime = ctx.GlobalDuration(TxPoolLifetimeFlag.Name)
 	}
+
+	// PN specific txpool setting
+	if ctx.GlobalIsSet(TxPoolSpamThrottlerDisalbleFlag.Name) {
+		blockchain.EnableSpamThrottlerAtRuntime = !ctx.GlobalBool(TxPoolSpamThrottlerDisalbleFlag.Name)
```

Comment:
Doesn't this always set `false`?

Reply:
EN/CN will return `false` for `ctx.GlobalIsSet(TxPoolSpamThrottlerDisalbleFlag.Name)`. So, this logic will not set `blockchain.EnableSpamThrottlerAtRuntime` for EN/CN.
PN will set `blockchain.EnableSpamThrottlerAtRuntime` by default.